app/src/main/java/com/google/android/samples/socialite/ui/videoedit/VideoEditScreenViewModel.kt

```diff
@@ -67,14 +65,19 @@ class VideoEditScreenViewModel @Inject constructor(
     private val _isProcessing = MutableStateFlow(false)
     val isProcessing: StateFlow<Boolean> = _isProcessing
 
+    private var _videoSaveState = MutableStateFlow(VideoSaveState.PENDING)
+    val videoSaveState: SharedFlow<VideoSaveState> = _videoSaveState
+
     fun setChatId(chatId: Long) {
         this.chatId.value = chatId
     }
 
     private val transformerListener: Transformer.Listener =
         @UnstableApi object : Transformer.Listener {
             override fun onCompleted(composition: Composition, exportResult: ExportResult) {
-                Toast.makeText(application, "Edited video saved", Toast.LENGTH_LONG).show()
+                viewModelScope.launch {
```

Comment:
Can these 3 lines be`_videoSaveState.value = VideoSaveState.VIDEO_SAVE_SUCCESS`? Or did it cause some issue?

Reply:
Good catch! Updated.